semantic_point_annotater fails to build on Kinetic

It looks to be missing a dependency.

http://build.ros.org/view/Kbin_uX64/job/Kbin_uX64__semantic_point_annotator__ubuntu_xenial_amd64__binary/14/console
```
fstack-protector-strong -Wformat -Werror=format-security -DNDEBUG -Wdate-time -D_FORTIFY_SOURCE=2     -DDISABLE_LIBUSB-1.0 -fPIC -o CMakeFiles/sac_inc_ground_removal_node.dir/src/sac_inc_ground_removal_standalone.cpp.o -c /tmp/binarydeb/ros-kinetic-semantic-point-annotator-0.1.27/src/sac_inc_ground_removal_standalone.cpp
<command-line>:0:15: warning: missing whitespace after the macro name
/tmp/binarydeb/ros-kinetic-semantic-point-annotator-0.1.27/src/sac_inc_ground_removal_standalone.cpp:43:21: fatal error: ros/ros.h: No such file or directory
compilation terminated.
```

--- a/semantic_point_annotator/CMakeLists.txt
+++ b/semantic_point_annotator/CMakeLists.txt
@@ -22,7 +22,7 @@ catkin_package(
 
 
 add_executable(sac_inc_ground_removal_node src/sac_inc_ground_removal_standalone.cpp src/sac/sac.cpp src/sac/ransac.cpp src/sac/sac_model.cpp src/sac/sac_model_line.cpp)
-include_directories(include/${PROJECT_NAME} ${Boost_INCLUDE_DIRS} ${PC_EIGEN_INCLUDE_DIRS} ${PCL_INCLUDE_DIRS}  ${caktin_INCLUDE_DIRS})
+include_directories(include/${PROJECT_NAME} ${Boost_INCLUDE_DIRS} ${PC_EIGEN_INCLUDE_DIRS} ${PCL_INCLUDE_DIRS}  ${catkin_INCLUDE_DIRS})
 target_link_libraries(sac_inc_ground_removal_node ${Boost_LIBRARIES} ${PCL_LIBRARIES} ${catkin_LIBRARIES})
 
 install(DIRECTORY include/${PROJECT_NAME}/

